python/mleap/sklearn/preprocessing/data.py

```diff
@@ -1097,8 +1098,12 @@ def fit_transform(self, X, y=None, **fit_params):
     def serialize_to_bundle(self, path, model_name):
         # compile tuples of model attributes to serialize
         attributes = list()
-        attributes.append(("labels", self.labels.keys()))
-        attributes.append(("values", Vector(self.labels.values())))
+        labels = (
+            sorted(self.labels.keys())
+            if not isinstance(self.labels, OrderedDict) else self.labels.keys()
```

Comment:
Does it make sense to save the self.labels as an OrderedDict once and for all ? Then we don't need to make sure we always sort or convert to OrderedDict everywhere else. Basically create one on line 1081

Reply:
sure, I'll update that :)